canonical-web-and-design/vanilla-framework
# Replace the github org with its new home

## Done
Replace the github org with its new home

## QA
- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check all the links are updated
